openfe/protocols/openmm_rfe/_rfe_utils/relative.py

```diff
@@ -1620,7 +1772,7 @@ def _check_indices(idx1, idx2):
                 # with zeroed out parameters; add old charge to
                 # standard_nonbonded and zero sterics
                 check_index = self._hybrid_system_forces['standard_nonbonded_force'].addParticle(
-                    charge_old, 0.5*(sigma_old+sigma_new), 0.0)
+                    charge_old, sigma_old, 0.0)
```

Comment:
I only just noticed this one.

Do we assert that sigma_old is equals to sigma_new elsewhere?